crates/services/block_aggregator_api/src/blocks/importer_and_db_source/serializer_adapter.rs

```diff
@@ -3,43 +3,119 @@ use crate::protobuf_types::V2Header as ProtoV2Header;
 use crate::{
```

Comment:
I think instead of creating many procedure fucntions, we could implement `TryFrom` and `From` for types, plus, we could split them into its own folders by transactions and some common folder.